File: 240204/폭탄 해제/bomb-removal.java
import java.io.*;
import java.util.*;

class Bomb {
    String code;
    char color;
    int time;
    public Bomb(String code, char color, int time) {
        this.code = code;
        this.color = color;
        this.time = time;
    }

    public void print() {
        System.out.println("code : " + this.code);
        System.out.println("color : " + this.color);
        System.out.println("second : " + this.time);
    }
}

public class Main {
    public static void main(String[] args) throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String[] input = br.readLine().split(" ");
        String code = input[0];
        char color = input[1].charAt(0);
        int time = Integer.parseInt(input[2]);
        new Bomb(code, color, time).print();
    }
}

File: 240317/폭탄 해제/bomb-removal.py
class Bomb:
    def __init__(self, code, color, time):
        self.code = code
        self.color = color
        self.time = time

code, color, time = input().split()
bomb = Bomb(code, color, int(time))
print(f'code : {bomb.code}')
print(f'color : {bomb.color}')
print(f'second : {bomb.time}')